sql/flaky-test-ratio.sql

```diff
@@ -0,0 +1,14 @@
+SELECT format_datetime(runs.created_at, 'Y:w') year_week
```

Comment:
Use a double `ww` so single digits get padded with zeros and get sorted in natural order.
```suggestion
SELECT format_datetime(runs.created_at, 'Y:ww') year_week
```